src/hubot.d.ts

```diff
@@ -60,8 +60,8 @@ declare module "hubot" {
     users(): { [id: string]: User; };
     userForName(name: string): User;
     userForId(id: string, options: any): User;
-    get(key: string): string;
-    set(key: string, value: string): Brain;
+    get(key: string): any;

```

Comment:
The brain can store anything, and the redis plugin does a `JSON.stringify` anyway. I do need to test this on a real redis brain.